runtime/js/02_console.js

```diff
@@ -1451,6 +1451,14 @@
       // by ObjectCreate(%ObjectPrototype%), instead of %ObjectPrototype%.
       const console = Object.create({});
       Object.assign(console, this);
+      // We define the property on the console object since the Object.assign
+      // does not copy the property definitions
+      Object.defineProperty(console, Symbol.toStringTag, {
+        enumerable: false,
+        writable: false,
+        configurable: true,
+        value: "console",
+      });
```

Comment:
Maybe merge the two statements into one?
```suggestion
const console = Object.create({}, {
    [Symbol.toStringTag]: {
      enumerable: false,
      configurable: true,
      value: "console",
    }
});

Object.assign(console, this);
```